whatlies/transformers/_tsne.py

```diff
@@ -51,6 +50,8 @@ def fit(self, embset):
 
     def transform(self, embset):
         names, X = embset.to_names_X()
+        # We are re-writing the transform method here because TSNE cannot .fit().transform().
+        # Check the docs here: https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html#sklearn.manifold.TSNE
         new_vecs = self.tfm.fit_transform(X)
         new_dict = new_embedding_dict(names, new_vecs, embset)
-        return EmbeddingSet(new_dict, name=f"{embset.name}.tsne_{self.n_components}()")
+        return EmbeddingSet(new_dict, name=f"{embset.name}.tsne()")
```

Comment:
Why removing the `n_components` from the name, unlike other transformers? Maybe you have meant writing `f"{embset.name}.{self.name}()"` instead?!